explorer/ast/static_scope.cpp

```diff
@@ -32,13 +38,19 @@ void StaticScope::MarkDeclared(std::string_view name) {
   CARBON_CHECK(it != declared_names_.end()) << name << " not found";
   if (it->second.status == NameStatus::KnownButNotDeclared) {
     it->second.status = NameStatus::DeclaredButNotUsable;
+    if (trace_stream_->is_enabled()) {
+      *trace_stream_ << "--- marked `" << name << "` declared but not usable\n";
```

Comment:
For this and the trace below, it'd be great to also include the name of the scope, once we start tracking that.